src/integrations/third-party/elementor.php

```diff
@@ -253,6 +253,7 @@ public function enqueue() {
 			\wp_enqueue_media( [ 'post' => $post_id ] );
 		}
 
+		$this->asset_manager->enqueue_style( 'admin-global' );
```

Comment:
Rather than enqueueing here we can also say that `elementor.css` depends on all of the styles implemented here and add those as a dependency (behind the `deps`=> key). Would have to check whether that works though, because I see it breaks with the pattern we were currently using. Not enough to block CR.

Reply:
@maartenleenders and I decided together to keep it this way as a lot of other styles are enqueued using the same approach.